docs/readthedocs/conf.py

```diff
@@ -0,0 +1,72 @@
+# Configuration file for the Sphinx documentation builder.
+#
+# This file only contains a selection of the most common options. For a full
+# list see the documentation:
+# https://www.sphinx-doc.org/en/master/usage/configuration.html
+
+# -- Path setup --------------------------------------------------------------
+
+# If extensions (or modules to document with autodoc) are in another directory,
+# add these directories to sys.path here. If the directory is relative to the
+# documentation root, use os.path.abspath to make it absolute, like shown here.
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'DWDWeatherTools'
+copyright = u'2022. TU Dortmund University, Institute of Energy Systems, Energy Efficiency and Energy Economics, Research group Distribution grid planning and operation '
+author = 'Johannes Hiry, Debopama Sen Sarma, Chris Kittl, Sebastian Peter, Thomas Oberließen'
+
+# The full version, including alpha/beta/rc tags
+version = '0.0.1'
+release = '0.1-SNAPSHOT'
```

Comment:
There is a divergence here. Does it make sense to resolve it?

Reply:
Is there? Isn't the current version being 0.0.1 with the next release being 0.1-SNAPSHOT reasonable?

Reply:
Wouldn't interpret it that way... 🤔 https://www.sphinx-doc.org/en/master/usage/configuration.html#confval-version